runtime/lua/vim/fs.lua

```diff
@@ -732,4 +732,28 @@ function M.abspath(path)
   return M.joinpath(cwd, path)
 end
 
+--- Return relative path from directory `base` to `target`. Note that `base` must be an ancestor to
+--- `target`, meaning relative paths starting with `../` are not supported.
+--- Example:
+---
+--- ```lua
+--- vim.fs.relpath('/var', '/var/lib') -- lib
```

Comment:
```suggestion
--- Gets `target` path relative to `base`, or `nil` if `base` is not an ancestor.
---
--- Example:
---
--- ```lua
--- vim.fs.relpath('/var', '/var/lib') -- 'lib'
```